.github/workflows/benchmark-command.yml

```diff
@@ -40,14 +40,14 @@ on:
           - 8h
           - 16h
           - 24h
+      folds:
+        description: 'Number of folds to run'
+        required: true
       branch_or_pr_number:
         description: 'Branch or PR number to run the benchmark on'
         required: true
-      fork_name:
-        description: 'Get the forked PR repository name'
-        required: true
-      fork_branch:
-        description: 'Get the forked PR repository branch name'
+      fork_info:
+        description: 'Get the forked PR repository name and branch'
```

Comment:
We shall add an example in the description

Reply:
Yes will do!

Reply:
Default fold value?

Reply:
default `fork_info`

Reply:
but I see now that `fork_info` is required, so no need I suppose.